tests/network/vmi_infosource.go

```diff
@@ -54,6 +58,15 @@ var _ = SIGDescribe("Infosource", func() {
 
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
+		fgDisabled := !checks.HasFeature(virtconfig.DynamicPodInterfaceNamingGate)
+		if fgDisabled {
+			config.EnableFeatureGate(virtconfig.DynamicPodInterfaceNamingGate)
+		}
+		DeferCleanup(func() {
+			if fgDisabled {
+				config.DisableFeatureGate(virtconfig.DynamicPodInterfaceNamingGate)
+			}
+		})
```

Comment:
This logic is uneeded.
Just enabled it, see [here](https://github.com/kubevirt/kubevirt/blob/d1a055b67783a852dbe1be519a6d32d0bee4c253/tests/network/bindingplugin.go#L50).

There is a test cleanup that restores KV to its original.

You also need to mark the test "serial" (see the ref one) because it touches the cluster level KV config.